### ISSUE TITLE
Provision a new database on a SQL-Pool

### ISSUE THREAD

**Issue by Herr-Sepp** [Azure/open-service-broker-azure#730]
Hi,
I found nothing in your documentation to create a new database within an existing database pool.
Is that even possible?

**Comment by zhongyi-zhang** [comment]
Pool -- do you mean a sql server manually created from Azure Portal or CLI? If so, please try the combination `azure-sql-12-0-dbms-registered` & `azure-sql-12-0-database-from-existing`: https://github.com/Azure/open-service-broker-azure/blob/master/docs/modules/mssql.md

**Comment by Herr-Sepp** [comment]
Sorry if I was too vague.

I mean the elastic SQL pool.
https://docs.microsoft.com/en-us/azure/sql-database/sql-database-elastic-pool

Can I create a new database within an existing pool using OSB?

**Comment by zhongyi-zhang** [comment]
No, OSBA doesn't support elastic SQL pool nor data warehouse. They needs different REST APIs and workflow from single SQL server to provision an instance. So we can't simply add service plans to support them. And there is no plan to support new features so far.

**Comment by Herr-Sepp** [comment]
Too bad.
Creating and managing pools are actually different APIs but I had hoped that creating a database within an existing pool is possible because it uses the same API "Database Create" as creating a normal single database.
https://docs.microsoft.com/rest/api/sql/databases/createorupdate
Request Body : properties.elasticPoolId

Is there a wish list or feedback hub for OBSA somewhere where I can request support for SQL pools?
Maybe a short sentence in the documentation that explain that pools are currently not supported would also be nice.


Thanks for your Support so far. 👍
